fabric-tree-api-v1/src/main/java/net/fabricmc/fabric/api/tree/v1/SimpleSaplingGenerator.java

```diff
@@ -0,0 +1,37 @@
+package net.fabricmc.fabric.api.tree.v1;
+
+import net.minecraft.block.sapling.SaplingGenerator;
+import net.minecraft.world.gen.feature.ConfiguredFeature;
+import net.minecraft.world.gen.feature.TreeFeatureConfig;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.Random;
+
+/**
+ * <p>A simple implementation of a {@link SaplingGenerator} returning the tree {@link ConfiguredFeature} given to it.</p>
+ * <p>Most of the time when working with {@link SaplingGenerator}s, you'll only need this functionality.</p>
+ * <p>
+ *     An example of using this implementation:<br>
+ *
+ *     <code>
+ *         public static final MY_SAPLING_GENERATOR = new SimpleSaplingGenerator(MY_TREE_CONFIGURED);
+ *     </code>
+ * </p>
+ */
+public class SimpleSaplingGenerator extends SaplingGenerator {
```

Comment:
The usefulness of this imo is limited; there is another `LargeTreeSaplingGenerator`, which this cannot cover apparently.

Reply:
I've removed this for now. Maybe will restore later.